### DISCUSSION TITLE
How can I disable keyboard during scroll

### DISCUSSION THREAD

**Discussion by flutterlovev** [community/community#29905] [Mobile] [answered]
How can I disable keyboard during scroll

**Comment by surajadkhari** [comment, marked as answer]
SingleChildScrollView or ListView, both widgets have a property called keyboardDismissBehavior. It represents how a ScrollView should dismiss the onscreen keyboard.

keyboardDismissBehavior:ScrollViewKeyboardDismissBehavior.onDrag
